torch/csrc/cpu/Module.cpp

```diff
@@ -16,6 +16,7 @@ void initModule(PyObject* module) {
   cpu.def("_is_amx_fp16_supported", at::cpu::is_amx_fp16_supported);
   cpu.def("_init_amx", at::cpu::init_amx);
   cpu.def("_is_arm_sve_supported", at::cpu::is_arm_sve_supported);
+  cpu.def("_is_arm_sve_bf16_supported", at::cpu::is_arm_sve_bf16_supported);
```

Comment:
Again, this looks like an anti-pattern. Perhaps a better API would be to return a named dict with all CPU properties...